package.json

```diff
@@ -113,7 +98,6 @@
     "vue-loader": "15.4.2",
     "vue-template-compiler": "2.5.21",
     "webpack": "4.28.3",
-    "webpack-bundle-analyzer": "3.0.3",
```

Comment:
I removed it by mistake, please leave it. Is convenient tool to analyze bundle sizes